src/main/java/org/apache/commons/math4/util/FastMath.java

```diff
@@ -2526,6 +2566,9 @@ public static double atan(double x) {
      * @param leftPlane if true, result angle must be put in the left half plane
      * @return atan(xa + xb) (or angle shifted by {@code PI} if leftPlane is true)
      */
+    @SuppressWarnings({"index:array.access.unsafe.low", "index:array.access.unsafe.high"}) /* #1: By #0.1, idx is always @NonNegative
```

Comment:
Then why `idx` is not annotated `@NonNegative`? Or rather `@IntRange(from=0, to=13)`?